core/drivers/crypto/crypto_api/drvcrypt_init.c

```diff
@@ -0,0 +1,59 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright 2018-2019 NXP
+ *
+ * Brief   This driver interfaces TEE Internal API by implementing
```

Comment:
"TEE Internal API" is an unfortunate wording as that's something different for us who have worked a bit with the GP APIs.

Reply:
ok I'll reword it.